File: algorithm/#0179/code.py
class Solution(object):
    def largestNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: str
        """
        numstr=map(str,nums)
        compare=lambda x,y: 1 if x+y<y+x else -1
        numstr.sort(cmp=compare)
        ret="".join(numstr)
        if ret[0]=="0":
            ret="0"
        return ret

File: algorithm/#1822/code.cpp
class Solution {
public:
    int arraySign(vector<int>& nums) {
        int ret=0;
        for(int i=0;i<nums.size();i++) {
            if(nums[i]==0) return 0;
            if(nums[i]<0) ret++;
        }
        if(ret%2) return -1;
        return 1;
    }
};
